modin/core/io/text/text_file_dispatcher.py

```diff
@@ -1115,3 +1113,48 @@ def _get_skip_mask(cls, rows_index: pandas.Index, skiprows: Callable):
             mask = rows_index.map(skiprows)
 
         return mask
+
+    @classmethod
+    @logger_decorator(
+        "PANDAS-API", "TextFileDispatcher._should_use_inferred_column_names", "INFO"
+    )
+    def _should_use_inferred_column_names(cls, names, skiprows, skipfooter, usecols):
+        """
+        Tells whether need to use inferred column names in workers or not.
+
+        1) ``False`` is returned in 2 cases and means next:
```

Comment:
Could you explain specifically why we have these particular checks on `skiprows` and `usecols` and `skipfooter`?

Reply:
An additional paragraph was added with explaining of the influence of parameters.